package.json

```
@@ -0,0 +1,68 @@
+{
+  "name": "fastify-secrets-hashicorp",
+  "version": "0.0.1",
+  "description": "Fastify secrets plugin for HashiCorp Vault",
+  "main": "lib/fastify-secrets-hashicorp.js",
+  "scripts": {
+    "lint": "eslint lib/**/*.js test/**/*.js",
```

I believe you can simply do:

```suggestion
    "lint": "eslint lib test",
```